src/server/constants.ts

```diff
@@ -1,11 +1,13 @@
+import { getSecret } from '../lib/secrets'
+
 export const PG_META_PORT = Number(process.env.PG_META_PORT || 1337)
-export const CRYPTO_KEY = process.env.CRYPTO_KEY || 'SAMPLE_KEY'
+export const CRYPTO_KEY = (await getSecret('CRYPTO_KEY')) || 'SAMPLE_KEY'
```

Comment:
Top-level await is only available in ESM. For one-off use cases like this using the sync functions should be fine.

Reply:
What's wrong with using ESM?

Reply:
Nothing wrong with it, but if we want it we should do the switch in another PR.

Reply:
> And our package.json also specifies the module type

Are you sure? It's not specified there, and iirc the default is `commonjs`.